langchain/chains/base.py

```diff
@@ -43,9 +44,21 @@ class Chain(BaseModel, ABC):
     """Base interface that all chains should implement."""
 
     memory: Optional[Memory] = None
+    callback_manager: Optional[BaseCallbackManager] = None
```

Comment:
can we do a default_factory like we do for verbose? eg `= Field(default_factory=get_callback_manager)`

honestly asking

Reply:
no idea, will try it out

Reply:
Ok so it works, but mypy is throwing a fit

Reply:
hmmm whats the error?